book/src/dev/zebra-dependencies-for-audit.md

```diff
@@ -0,0 +1,150 @@
+# Zebra dependencies
+
+This is a list of Rust code that is in scope and out of scope for Zebra's first audit:
+- **bold**: Crates that we want to be audited fully.
+- <i>italic</i>: Crates we want to be audited with conditions.
```

Comment:
```suggestion
This is a list of production Rust code that is in scope and out of scope for Zebra's first audit:
- **bold**: Crates that we want to be audited fully.
- <i>italic</i>: Crates we want to be audited with conditions.

Test code, deployment configurations, and other configuration files in the `zebra` repository are out of scope.
```

Reply:
I applied this in https://github.com/ZcashFoundation/zebra/pull/5471

Reply:
I applied this in https://github.com/ZcashFoundation/zebra/pull/5471